container-specs.md

```diff
@@ -0,0 +1,19 @@
+Container Specifications
+========================
+
+Each container should provide a well-defined metadata that allows the final users to test, deploy and get support around the container. 
+We explain here the containers metadata, their meaning and use in BioContainers: 
+ 
+
+
+| Field                 | Description                                                                                                                | Example                                                  | 
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| software version      | Every Biocontainer should contains the actual version of the software and tool                                             | software version: 2015020                                | 
+| software              | Name of the software                                                                                                       | software: Comet                                          |
+| description           | A short description of the software or tool.                                                                               | description: Peptide identification search tool          | 
```

Comment:
This makes all a lot of sense for regular images but less sense for mulled based ones. For example the version and the name is encoded in the namespace, the base-image is always `busybox` and there is no real maintainer.
That said we can think about adding these informations as yaml file into the container as `/usr/share/biocontainer_info.yaml`?

Also I would like to point out that mulled based containers are automatically build so all annotations we need to add manually would destroy one very nice feature of these containers :)

Here, I would like to propose to annotate the original recipe (conda, alpine, brew ...) and include the original recipe in the container as metadata.